core/server/models/plugins/order.js

```diff
@@ -39,6 +42,15 @@ const order = function order(Bookshelf) {
                 field = match[1].toLowerCase();
                 direction = match[2].toUpperCase();
 
+                if (orderRawAttributes && orderRawAttributes[field]) {
+                    const {orderByRaw, eagerLoad} = orderRawAttributes[field];
+                    orderRaw.push(`${orderByRaw} ${direction}`);
```

Comment:
Thinking it would probably be better to pass direction to `this.orderRawAttributes()` because appending it here is unlikely to work with all possible raw order queries.

Reply:
How would the call to `this.orderRawAttributes()` look like? Would we be storing full `orderRaw` value including order inside the orderRawAttributes?

Reply:
```js
    orderRawAttributes: function orderRawAttributes(withRelated, direction) {
        if (withRelated && withRelated.indexOf('email') > -1) {
            return {
                'email.open_rate': {
                    orderByRaw: `emails.opened_count / emails.email_count * 100 ${direction}`,
                    eagerLoad: 'email.open_rate'
                }
            };
        }
    },
```

Reply:
It's something we can change in the future if/when needed though. I was thinking ahead to having multi-clause order by queries but it's an edge case, not needed for our current use-cases.